creator/ingest_runs/queries/validation_run.py

```diff
@@ -24,14 +27,40 @@ class Query(object):
         filterset_class=ValidationRunFilter,
         description="Get all validation_runs",
     )
+    last_modified_active_validation_run = Field(
+        ValidationRunNode,
+        description=(
+            "Get the last modified active validation_run for a data_review"
+        ),
+        dr_id=String(required=True),
+    )
 
     def resolve_all_validation_runs(self, info, **kwargs):
         """
         Return all validation_runs
         """
         user = info.context.user
-
         if not user.has_perm("ingest_runs.list_all_validationrun"):
             raise GraphQLError("Not allowed")
 
         return ValidationRun.objects.all()
+
+    def resolve_last_modified_active_validation_run(
+        self, info, dr_id, **kwargs
+    ):
+        """
+        Return the last modified validation_run for the data_review with
+        kf_id=dr_id which is currently active (i.e. not completed, canceled, or
+        failed).
+        """
+        user = info.context.user
+        if not user.has_perm("ingest_runs.list_all_validationrun"):
```

Comment:
This is something I'm not sure of. Should we add a new permission for this query?